posthog/api/organization_feature_flag.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework.views import APIView
+from posthog.models import FeatureFlag
+from posthog.models.organization import Organization
+from rest_framework.response import Response
+from posthog.permissions import OrganizationMemberPermissions
+from rest_framework.permissions import IsAuthenticated
+from rest_framework import serializers
+
+
+class OrganizationFeatureFlagView(APIView):
```

Comment:
I was a bit mistaken earlier on APIViews - looking at the docs again, these are for single endpoints, while ViewSets are for adding multiple endpoints in the same place.

As such, I think what we need here is a `ViewSet`, (in contrast to the existing `ModelViewSet` which require a queryset - viewset does not and gives us all we need: https://www.django-rest-framework.org/api-guide/viewsets/

Instead of `list()` and `retrieve()` you'll just have @action decorators for the two new functions: view flags & update flag

https://www.django-rest-framework.org/api-guide/viewsets/#marking-extra-actions-for-routing

<img width="855" alt="image" src="https://github.com/PostHog/posthog/assets/7115141/c8089328-62e7-4896-accf-250255799456">